src/apt/supply/supply.go

```diff
@@ -106,10 +124,28 @@ func (s *Supplier) createSymlinks() error {
 		if exists, err := libbuildpack.FileExists(dest); err != nil {
 			return err
 		} else if exists {
-			if err := s.Stager.LinkDirectoryInDepDir(dest, dirs[1]); err != nil {
+			files, err := ioutil.ReadDir(dest)
+			if err != nil {
 				return err
 			}
+			destDir := filepath.Join(s.Stager.DepDir(), dirs[1])
+			if err := os.MkdirAll(destDir, 0755); err != nil {
+				return err
+			}
+			for _, file := range files {
+				//TODO: better way to copy a file?
```

Comment:
Libbuildpack provides a utility for copying files (source here: https://github.com/cloudfoundry/libbuildpack/blob/6d1936c34b28898171653fb8d2dfca1144499095/util.go#L160) that you could use: `libbuildpack.CopyFile(..,..)`

Reply:
Ah, I may have jumped the gun; I see you're also editing the contents.